packages/dx-chart-core/src/plugins/tooltip/computeds.js

```diff
@@ -0,0 +1,8 @@
+export const getParameters = (series, target) => {
+  if (target) {
+    const currentSeries = series.find(({ name }) => target.series === name);
+    const item = currentSeries.points[target.point];
```

Comment:
`point` is index within `data` array (particular series can have less points then items in `data` array)

Use `points.find(point => point.index === index)` or `d3.bisect`.